tests/test_example_client.py

```diff
@@ -40,7 +40,7 @@ def mock_connect_after_rejections(self, *args, **kwargs):
     self.connected = True
 
 
-def mock_start_stream(self, media_generator):
+def mock_start_stream(self, media_generator, media_config, response_types):
```

Comment:
This fixed the unit-tests, we should somehow start running this in CI somewhere.